gotham/Cargo.toml

```diff
@@ -32,6 +33,10 @@ linked-hash-map = "0.5"
 num_cpus = "1.8"
 crossbeam = "0.3"
 regex = "1.0"
+cookie = "0.10"
+http = "0.1.7"
```

Comment:
I think it'd be better if we stuck to `x.x` minor for packages, so that:

```
http = "0.1"
failure = "0.1"
failure_derive = "0.1"
```

Just typically easier to pick up bugfixes that way.

Reply:
My understanding was that `x.y.z` -> `^x.y.z` -> `>= x.y.z, <x.y+1` For semvered crates, that should cover bugfixes without added features. Of course, for `x=0` all bets are off anyway.

Is my understanding right here? I can definitely see the logic to relaxing to `0.y` even if so, though.